SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/AccountsView.kt

```diff
@@ -0,0 +1,541 @@
+package app.cybrid.sdkandroid.components
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.activity.compose.BackHandler
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import app.cybrid.cybrid_api_bank.client.models.TradeBankModel
+import app.cybrid.sdkandroid.R
+import app.cybrid.sdkandroid.components.accounts.entity.AccountAssetPriceModel
+import app.cybrid.sdkandroid.components.accounts.view.AccountsViewModel
+import app.cybrid.sdkandroid.components.listprices.view.ListPricesViewModel
+import app.cybrid.sdkandroid.core.Constants
+import app.cybrid.sdkandroid.ui.Theme.robotoFont
+
+class AccountsView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
```

Comment:
Wonder if we could use an Enum instead of Int?

Reply:
Maybe, you can propose this to Google, this defStyle its defined and used in ConstraintLayout, this are the neccesary params of Google ask in the View Protocol.

AccountView heredate from Component and this heredate from ConstraintLayout

```
open class Component @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyle: Int = 0
) : ConstraintLayout(context, attrs, defStyle)
```

Reply:
ConstriantLayout definition:

`    public ConstraintLayout(@androidx.annotation.NonNull android.content.Context context, @androidx.annotation.Nullable android.util.AttributeSet attrs, int defStyleAttr) { /* compiled code */ }
`

Reply:
Second the defStyle its a id reference, like `R.theme.light` its not a type enum

Reply:
I mean for better maintainability, you could have a custom enum with Int Raw Type, and you can pass the raw value to the Component parent. I leave it to you.